From ace9070b6f9a57f96879d2c1476687e892e2de8e Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 11 Feb 2024 19:06:45 +0100
Subject: [PATCH 37/48] 8723cs: Ignore warnings

It's getting too noisy with the latest gcc/linux flags.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/staging/rtl8723cs/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/rtl8723cs/Makefile b/drivers/staging/rtl8723cs/Makefile
index 7b630a48af64..9e8ca7cefe1d 100644
--- a/drivers/staging/rtl8723cs/Makefile
+++ b/drivers/staging/rtl8723cs/Makefile
@@ -9,6 +9,7 @@ EXTRA_CFLAGS += -O2
 #EXTRA_CFLAGS += -Wshadow -Wpointer-arith -Wcast-qual -Wstrict-prototypes -Wmissing-prototypes
 
 EXTRA_CFLAGS += -Wno-unused-variable
+EXTRA_CFLAGS += -Wno-missing-prototypes -Wno-missing-declarations -Wno-stringop-overread -Wno-overflow -Wno-address
 #EXTRA_CFLAGS += -Wno-unused-value
 #EXTRA_CFLAGS += -Wno-unused-label
 #EXTRA_CFLAGS += -Wno-unused-parameter
-- 
2.43.0

